doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,12 @@
 
 <h3>New features since last release</h3>
 
+* Continued development of circuit cutting compiler:
+  
+  A method to replace `WireCut` nodes in a directed multigraph with `MeasureNode` and
+  `PrepareNode` placeholders has been added.
+  [(#2124)](https://github.com/PennyLaneAI/pennylane/pull/2124)
+  
```

Comment:
Maybe we should move the entry for [2107](https://github.com/PennyLaneAI/pennylane/commit/ced3499b4043f66d5a6f303fdd0a594ffe8deb3f#diff-e6ab4fddf160607572464ae0724adc6cf2cb437959a8f238382eb0d0befc92f7R55) to here

Reply:
Added thanks! Also realised that the `2107` entry may have snuck into the wrong change log since it won't be included in this release. Should I remove this from `changelog-0.21.0.md`?

Reply:
Sounds good, thanks!